src/pku2u/mod.rs

```diff
@@ -380,11 +378,33 @@ impl SspiImpl for Pku2u {
         })
     }
 
-    #[instrument(ret, fields(state = ?self.state), skip_all)]
+
+
+    #[instrument(level = "debug", ret, fields(state = ?self.state), skip(self, _builder))]
+    fn accept_security_context_impl<'a>(
+        &'a mut self,
+        _builder: crate::builders::FilledAcceptSecurityContext<'a, Self::AuthenticationData, Self::CredentialsHandle>,
+    ) -> Result<AcceptSecurityContextResult> {
+        Err(Error::new(
+            ErrorKind::UnsupportedFunction,
+            "accept_security_context_impl is not implemented yet",
+        ))
+    }
+    
+    // #[instrument(ret, fields(state = ?self.state), skip_all)]
```

Comment:
Move this instrument annotation at the appropriate place

Reply:
updated